.storybook/main.js

```diff
@@ -22,7 +21,6 @@ module.exports = {
       ...config,
       plugins: [
         ...config.plugins,
-        ...webpackConfig.plugins,
```

Comment:
If we no-longer need plugins from our webpack.config, then we can remove the logic that detects the ENV and conditionally generates the group of plugins. We can just straight define plugins as whatever is needed for the production build.

Reply:
@JCarmine good call. We still need to keep the css modules config rule but everything else is now outside of the conditional statement.